unixfs/tar/reader.go

```diff
@@ -6,6 +6,7 @@ import (
 	"compress/gzip"
 	"io"
 	p "path"
+	"strings"
```

Comment:
know its not part of this pr, but can you change `p` to something else? like `gopath` or something? `p` is not very clear.